### ISSUE TITLE
[OSSPlanet] CinelarTV - Application for VPS Plan

### ISSUE THREAD

**Issue by yanquisalexander** [OSSPlanet/OSSPlanet#15]
## Applicant:
[CinelarTV](https://github.com/CinelarTV/core)

![circle-cropped,128x128](https://user-images.githubusercontent.com/53713818/79126971-56db1500-7d77-11ea-806e-8cfec7884363.png)

---

## Project Info:
CinelarTV is a project created by two high school students, we want to experience on our own how it is to manage a video streaming service on demand.
Our core code is available in the previously mentioned repository (CinelarTV/core)

## Contact:
[Alexander Barrios on Telegram](https://t.me/alexander_291004)

## Who I am:
My name is Alexander, I am a 15 year old student, I wanted to create CinelarTV as a challenge to know for myself what it feels like to manage a video on demand platform, also to learn about the creation of Mobile Applications and Web development

## Verification:
We do not have an official site yet, but I quote links where I have published about CinelarTV

- https://github.com/orgs/CinelarTV/people
- https://community.kodular.io/t/cinelartv-a-digital-streaming-platform/45346

Our Discourse Community (Hosted on Discourse for OpenSource):

- https://cinelartv.discourse.group

## Resource you need:
Basically we would need 2 VPS with LEMP, one would host the Api gateway and the other the core

## Any additional info.

We are open to questions

## Ability to promote OSSPlanet

To promote OSSPlanet we can include attributions in our application, in our future website, and in our posts in Community, in addition, OSSPlanet members who register in CinelarTV Community will be able to access a special private category

**Comment by bluet** [comment]
Hello @yanquisalexander  could you also send a mail to apply ossplanet net and bluet bluet org. http://ossplanet.net/#apply
